qCC/ccSetClassificationField.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CCSETCLASSIFICATIONFIELD_H
+#define CCSETCLASSIFICATIONFIELD_H
+
+#include <QDialog>
+
+
+namespace Ui {
+class ccSetClassificationField;
+}
+
+//namespace ccHObject{
+//class Container;
+//}
+
+#include <ccHObject.h>
+
+class ccSetClassificationField : public QDialog
```

Comment:
It would be good to add Dlg or Dialog at the end --> ccSetClassificationFieldDialog